resources/gin/prepare/tracknet_v2_prepare.cfg

```diff
@@ -1,11 +1,11 @@
-preprocess.target_processor = @TrackNet_Processor
-preprocess.output_dir = 'output/cgem_t_tracknet'
+preprocess.target_processor = @TrackNetProcessor
+preprocess.output_dir = 'output/cgem_t_tracknet_new'
 preprocess.ignore_asserts = True
 
-parse.input_file_mask='data/bes3/events/1.txt'
+parse.input_file_mask='data/new_data/tracknet_3.txt'
 parse.csv_params={
                     "sep": '\s+',
-                    "nrows": 1000,
+                    "nrows": 10000000,
```

Comment:
we may simply comment this line and the whole file will be parsed

Reply:
Thanks, fixed